src/main/cms/cms_menu_vtx_tramp.c

```diff
@@ -67,16 +70,24 @@ void trampCmsUpdateStatusString(void)
     }
     trampCmsStatusString[4] = ' ';
 
-    if (trampCurFreq)
-        tfp_sprintf(&trampCmsStatusString[5], "%4d", trampCurFreq);
-    else
+    uint16_t freq;
+    if (!vtxCommonGetFrequency(vtxDevice, &freq) || (freq == 0))
         tfp_sprintf(&trampCmsStatusString[5], "----");
-
-    if (trampPower) {
-        tfp_sprintf(&trampCmsStatusString[9], " %c%3d", (trampPower == trampConfiguredPower) ? ' ' : '*', trampPower);
+    else
+        tfp_sprintf(&trampCmsStatusString[5], "%4d", freq);
+
+    uint16_t actualPower = vtxTrampGetCurrentActualPower();
+    uint8_t powerIndex;
+    uint16_t powerValue;
+    if (    actualPower > 0
```

Comment:
Please remove extra spaces.

Reply:
I've flattened this statement out onto one line. I wasn't sure if you were referring to the spaces before actualPower or the multiple lines. The spaces are a throwback I find hard to resist from a former employer, lining up all the expressions within the if statement. Let me know if you'd prefer I wrap the statements again. That line's coming in at just over 140 characters at present.

Reply:
In general, the aim of the coding style in Betaflight is to make the code easy to maintain as the community driven open source project that it is. One part of this is to aim for formatting that will result in only lines with actual functional change showing up in future reviews. Any form of artificial alignment between lines that will need re-alignment of multiple lines sooner or later is therefore discouraged.